### PR TITLE
Update pom.xml in preparation of ADO pipeline

### PR DESCRIPTION
**Pull request by ivanTrogrlic**

### CODE REVIEW

**Comment by ivanTrogrlic** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: pom.xml [hunk 1]
```diff
@@ -49,25 +49,8 @@ SOFTWARE
...
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
-  </distributionManagement>
-
   <properties>
+    <project.version>${project.version}</project.version>
```
Comment (abyss638) [inline pom.xml:53]: Is it really needed?

File: .config/tsaoptions.json
Comment (abyss638) [inline .config/tsaoptions.json:1]: I don't think this file needed?

File: pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@ SOFTWARE
 
   <groupId>com.microsoft</groupId>
   <artifactId>credential-secure-storage</artifactId>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>1.0.3</version>
```
Comment (abyss638) [inline pom.xml:30]: Please update version in sample pom.xml

**Review by abyss638 — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml
Comment (abyss638) [inline azure-pipelines.yml:1]: Please delete deploy.yml file in workflows as well

File: pom.xml (hunk 1 above)
Comment (ivanTrogrlic) [reply pom.xml]: Removed

File: .config/tsaoptions.json
Comment (ivanTrogrlic) [reply .config/tsaoptions.json:1]: Removed

File: pom.xml (hunk 2 above)
Comment (ivanTrogrlic) [reply pom.xml:30]: Updated

File: azure-pipelines.yml
Comment (ivanTrogrlic) [reply azure-pipelines.yml:1]: Removed

**Pull request merged**